src/game.py

```diff
@@ -0,0 +1,61 @@
+import random
+from dataclasses import dataclass
+
+
+@dataclass
+class Game:
+    """Class for overall game."""
```

Comment:
We can expand this docstring at the end of the process. "Class for a tournament style, dueling game between fictional characters from the Harry Potter franchise. " is a suggestion.